client/src/components/controls/CustomEvent.tsx

```diff
@@ -240,9 +240,14 @@ const CustomEvent: React.FC = () => {
           horizontal: 'right',
         }}
         transformOrigin={{
-          vertical: 'bottom',
+          vertical: 'top',
           horizontal: 'right',
         }}
+        PaperProps={{
+          style: {
+            minWidth: createEventAnchorEl?.clientWidth || undefined,
+          },
+        }}
```

Comment:
PaperProps is deprecated. Use this instead

```suggestion
        slotProps={{
          paper: {
            style: {
              minWidth: createEventAnchorEl?.clientWidth,
            },
          },
        }}
```